03-Quotes (copy)/quotes.js

```diff
@@ -0,0 +1,146 @@
+// class MyQuote{
+//   constructor()
+
+
+// }
+
+
+const randomQuotes=document.getElementById('randomQuotes').addEventListener('click',renderQuotes("random"))
+window.addEventListener('load',(e)=>{
+  renderQuotes()
+})
+let listWrapper = document.getElementById('listWrapper')
+
+// const wisdomQuotes=document.querySelector("input[name=wisdom]")
+
+// wisdomQuotes.addEventListener( 'click', function() {
+//   if(this.checked) {
+//     //  console.log('Checkbox is checked..') 
+//      renderWisdom()
+//   } else {
+//     console.log('Checkbox is not checked..')   
+//   }
+// });
+// let wisdomlist=document.getElementById('wisdomlist')
+
+async function wisdomQuote(){
+  try{
+    let resp=await fetch('https://api.quotable.io/quotes?tags=wisdom')
+    return await resp.json()
+  }catch(error){
+    console.log(error)
+  }
+}
+async function renderWisdom() {
+  const renderWisdom = await wisdomQuote();
+  let wlist=''
+   renderWisdom.results.forEach(results => {
+      wlist+= `<div>
+      <span>${results.tags}</span>
+      <span>${results.content}</span>
+      <span>${results.author}</span>
+      </div>`;
+});
+
+ 
+list.innerHTML = wlist;
+}
+
+
+async function getQuotes() {
```

Comment:
as mentioned in the call yesterday break down your functions and have separate functions for each operation so as to reuse the code